examples/sdp_examples/Converting a non-dcp compliant problem to Semi-Definite Programming Problem.ipynb

```diff
@@ -0,0 +1,111 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#  Non-DCP Compliant Problem to Semi-Definite Programming Problem \n",
+    "\n",
+    "Consider the problem\n",
+    "\n",
+    "minimize $\\frac{(c^T x)^2}{d^Tx}$\n",
```

Comment:
I'd suggest writing optimization problems with equation arrays in latex:

\begin{array}{ll}
\mbox{minimize} & \frac{(c^T x)^2}{d^Tx} \\
\mbox{subject to} & \ldots
\end{array}